nvidia/lib

```diff
@@ -9,6 +9,12 @@ ARCH_TRIPLET="${U_MACHINE}-${U_KERNEL,,}-${U_USERLAND,,}"
 
 NVIDIA_SUPPORT_DISABLED="$(snapctl get nvidia-support.disabled)"
 
+if [ -L /var/lib/snapd/hostfs/usr/lib/x86_64-linux-gnu/libcuda.so ] ; then
```

Comment:
This should be made architecture-agnostic:
```suggestion
if [ -L /var/lib/snapd/hostfs/usr/lib/"${ARCH_TRIPLET}"/libcuda.so ] ; then
```
Note that I haven't tested this.

Reply:
Yes, good point, I've altered that to similar.